### DISCUSSION TITLE
请问发送图片可以不压缩直接发原图吗

### DISCUSSION THREAD

**Discussion by Arondight** [takayama-lily/oicq#244] [Q&A] [answered]
图一压就不能看了 https://github.com/Arondight/Adachi-BOT/issues/234#issuecomment-949311306

**Comment by Arondight** [comment]
@takayama-lily 大佬方便回答一下吗

**Comment by takayama-lily** [comment, marked as answer]
上传的图本来就不压缩，只是客户端看到的是压缩过的。用这个地址可以访问到原图：
https://gchat.qpic.cn/gchatpic_new/0/0-0-MD5/0
对于jpg是这样，png貌似是直接给原图。

**Reply by Arondight** [reply]
谢谢解答

**Comment by YTheSeven** [comment]
挖个坟，手机qq有个发送原图的选项，api中没有这个配置嘛QAQ

**Reply by YTheSeven** [reply]
@takayama-lily 佬救一下

**Reply by xueelf** [reply]
> @takayama-lily 佬救一下

高山不是都说了图片不会压缩，你不想发原图自己压一遍呀

**Reply by YTheSeven** [reply]
> 

是想发原图别的地方看得不是原图...

**Comment by takayama-lily** [comment]
ImageElem中有个origin属性用来显示下载原图按钮
